dist/ibantools.js

```diff
@@ -151,6 +177,28 @@ define(["require", "exports"], function (require, exports) {
         return false;
     }
     exports.isSEPACountry = isSEPACountry;
+    /**
+     * Check if IBAN is QR-IBAN
+     * ```
+     * // returns true
+     * ibantools.isQRIBAN("CH4431999123000889012");
+     * ```
+     * ```
+     * // returns false
+     * ibantools.isQRIBAN("NL92ABNA0517164300");
+     * ```
+     */
+    function isQRIBAN(iban) {
+        if (iban === undefined || iban === null)
+            return false;
+        var countryCode = iban.slice(0, 2);
+        var QRIBANCountries = ['LX', 'CH'];
```

Comment:
Should be
`var QRIBANCountries = ['LI', 'CH'];`
LI - Liechtenstein
CH - Switzerland